csharp-intermediate-topics/WhenToUseReaderWriterLockSlimOverLockInCSharp/WhenToUseReaderWriterLockSlimOverLockInCSharp/BaseReaderWriter.cs

```diff
@@ -1,49 +1,48 @@
-﻿using System.Diagnostics;
+﻿namespace WhenToUseReaderWriterLockSlimOverLockInCSharp;
 
-namespace WhenToUseReaderWriterLockSlimOverLockInCSharp
+using System.Diagnostics;
+
+public abstract class BaseReaderWriter
 {
-    public abstract class BaseReaderWriter
+    protected readonly List<int> NumbersList = new();
+
+    public List<int> ListOfNumbers => NumbersList;
+
+    public long Execute(ThreadExecutionConfiguration config)
     {
-        protected readonly List<int> _listOfNumbers = new();
+        var tasks = new List<Task>();
```

Comment:
This is a minor thing, but since we know the size, we should initialize the list with a count